packages/perseus-score/src/widgets/expression/score-expression.test.ts

```diff
@@ -67,6 +67,16 @@ describe("scoreExpression", () => {
         expect(result).toHaveInvalidInput();
     });
 
+    it("should handle correct answers with colon divide symbol", function () {
+        const result = scoreExpression("z:3", expressionItem3Options, "uk");
+        expect(result).toHaveBeenAnsweredCorrectly();
+    });
+
+    it("should handle invalid answers with colon divide symbol", function () {
+        const result = scoreExpression("z:3", expressionItem3Options, "en");
+        expect(result).toHaveInvalidInput();
+    });
+
```

Comment:
It doesn't look like there are any tests for the en divide sign in this file? Could it be worth adding one? I know it's already written as `z/3` in the test data, but I think it could be worth confirming that it still works in the `z/3` case.